heapoverflow/heapoverflow.json

```
@@ -0,0 +1,24 @@
+{
+    "key": "HeapOverflow",
+    "label": "HeapOverflow",
+    "display": "standalone",
+    "access": "external",
+    "image": {
+        "name": "heapoverflow",
+        "tags": [
+            "latest"
+        ]
+    },
+    "logo": "",
+    "info": "",
```

Linking back to the documentation for this image:
```suggestion
    "info": "https://github.com/cheese-hub/Heap-Overflow",
```